src/NNCache.cpp

```diff
@@ -77,7 +77,7 @@ void NNCache::set_size_from_playouts(int max_playouts) {
     // cache hits are generally from last several moves so setting cache
     // size based on playouts increases the hit rate while balancing memory
     // usage for low playout instances. 150'000 cache entries is ~225 MB
-    auto max_size = std::min(150'000, std::max(6'000, 3 * max_playouts));
+    auto max_size = 3 * std::min(50'000, std::max(2'000, max_playouts));
```

Comment:
Not clear why this is being changed. It's awkward now because the 150`000 from the comment is now implicit.

Reply:
It is avoiding the potential overflow when `-p` isn't used and the `cfg_max_playouts` is left defaulted to int max (or `-p` is used and anything at least int max / 3 is specified). It seemed awkward to test the int max / 3 limit only to then limit the result again. Thinking about it again, I agree that it is clearer to have the max limit written in code to match the comment, in terms of as cache entries rather than as playouts, so I'll just do it the "awkward" way.